.github/workflows/pull-request-management.yml

```diff
@@ -204,10 +204,13 @@ jobs:
           - 'example-single-dc-l3ls'
         ansible_version:
           - 'ansible-core<2.17.0 --upgrade'
+        pip_requirements:
+          - 'ansible_collections/arista/avd/requirements.txt'
         # Also test minimum ansible version for one scenario.
         include:
           - avd_scenario: 'eos_designs_unit_tests'
             ansible_version: 'ansible-core==2.14.0'
+            pip_requirements: 'ansible_collections/arista/avd/requirements-minimum.txt'
```

Comment:
Hmm so it looks like it was needed .. and not picking up the default vairable .. my bad..